test/functional/page_objects/time_picker.ts

```diff
@@ -196,7 +192,6 @@ export class TimePickerPageObject extends FtrService {
       await this.testSubjects.click('querySubmitButton');
     }
 
-    await this.waitPanelIsGone(panel);
```

Comment:
@achyutjhunjhunwala did you intentionally remove these?

Reply:
Yes, as this was adding no value but more flakiness.